src/components/Card.tsx

```diff
@@ -88,15 +96,14 @@ const cardContainer = (
       'py-5': paddingType === 'normal',
       'py-8': typeof paddingType === 'undefined',
     }),
-    width(
-      thin ? 'sm:!w-thin-card' : 'sm:w-card',
-      thin ? 'tiny:w-thin-mobile' : 'w-mobile-card',
-      { 'w-32': thin }
-    ),
+    !useAppStyles ? (thin ? thinWidthStyles : defaultWidthStyles) : undefined,
```

Comment:
Don't convert `!useAppStyels`, do:
```TS
useAppStyles ? undefined : (thin ? ... : ... )
```

Reply:
Fewer conversions = easier reading + faster code

Reply:
Oh I like this change, thanks